tests/integration_tests/conftest.py

```diff
@@ -82,12 +98,14 @@ async def lobby_server(
     rating_service,
     message_queue_service,
     party_service,
-    policy_server
+    oauth_service,
+    policy_server,
+    jwks_server
 ):
-    with mock.patch(
-        "server.lobbyconnection.config.FAF_POLICY_SERVER_BASE_URL",
-        f"http://{policy_server.host}:{policy_server.port}"
-    ):
+    with mock.patch("server.lobbyconnection.config.FAF_POLICY_SERVER_BASE_URL",
```

Comment:
Oh I see what's going on here, two context managers in the same with statement. I didn't read that correctly last time. Using `\` line continuation is definitely a red flag for some possibly bad formatting. Maybe it would be better to assign the context managers to variables:
```python3
patch_policy = mock.patch(...)
patch_jwks = mock.patch(...)

with patch_policy, patch_jwks:
    ...
```

Reply:
Done